README.md

```diff
@@ -7,13 +7,6 @@ FastBoot allows you to render and serve Ember.js apps on the server.
 Using FastBoot, you can serve rendered HTML to browsers and
 other clients without requiring them to download JavaScript assets.
 
-Currently, the set of Ember applications supported is extremely limited.
-As we fix more issues, we expect that set to grow rapidly. See [Known
-Limitations](#known-limitations) below for a full-list.
-
-The bottom line is that you should not (yet) expect to install this add-on in
```

Comment:
This sentence is actually still true, or at least how I am reading it: "you can't expect to install this and have everything 'just work' and you will likely need to tweak some things"

Consider rephrasing rather than removing 👍